app/commands/git/sync_concept.rb

```diff
@@ -48,7 +48,7 @@ def concept_config_modified?
 
     memoize
     def concept_config
-      # TODO: determine what to do when the concept could not be found
+      # TODO: (Optional): determine what to do when the concept could not be found
```

Comment:
I made this optional as we'll automatically create a sync issue on GitHub if this happens.